src/test/store/profile-view.test.js

```diff
@@ -9,7 +9,10 @@ import * as ProfileView from '../../actions/profile-view';
 import { selectedThreadSelectors } from '../../reducers/profile-view';
 
 describe('call node paths on implementation filter change', function() {
-  const { profile, funcNames } = getProfileFromTextSamples(`
+  const {
+    profile,
+    funcNamesPerThread: [funcNames],
```

Comment:
There was a lot of code churn with changing this function's returning `funcNames` value to be per thread.